packages/model-viewer/src/features/scene-graph/api.ts

```diff
@@ -39,7 +43,29 @@ export declare interface Model {
    * correspond to the listing of materials in the glTF, with the possible
    * addition of a default material at the end.
    */
-  readonly materials: Readonly<Array<Material>>;
+  readonly materials: Readonly<Material[]>;
+
+  /**
+   * Gets a material(s) by name.
+   * @param name the name of the material to return.
+   * @returns a list of materials who's name is equal to `name` or an empty list
```

Comment:
nit: whose name?

Reply:
lol, thank you, I knew that felt wrong :D